adserver/admin.py

```diff
@@ -770,6 +780,13 @@ def refund_impressions(self, request, queryset):
         return None
 
 
+class OfferAdmin(AdBaseAdmin):
+
+    """Django admin configuration for ad offers."""
+
+    model = Offer
```

Comment:
Should we add `viewed` and `clicked` here? Might be useful in dev but probably not very useful in prod.

Reply:
Makes sense :+1: